new-client/src/plugins/DocumentHandler/printMenu/PrintWindow.js

```diff
@@ -840,30 +903,19 @@ class PrintWindow extends React.PureComponent {
     );
   };
 
-  render() {
-    const { togglePrintWindow, localObserver, documentWindowMaximized } =
-      this.props;
+  toggleDocumentsAttachments = () => {
+    let showAttachments = this.state.showAttachments;
+    this.setState({ showAttachments: !showAttachments });
+  };
```

Comment:
This could lead to some state-timing issues, i would suggest:

```js
this.setState((prevState) => ({ showAttachments: !prevState.showAttachments }));

Reply:
Fixed!